build.sh

```diff
@@ -379,47 +405,102 @@ build_jdk() {
         *)
             unknown_platform;;
     esac
-    chmod -R 755 ${wrkdir}/openjdk/build || exit $?
+    # JDK installs some jar files unreadable to "other" users meaning that the
+    # benchmark user can't access them. This becomes a problem later for graal,
+    # which takes a copy of this JDK's jar files.
+    chmod -R 755 ${wrkdir}/openjdk/build/*-release/jdk/lib || exit $?
+}
+
+
+# This is a bootstrap JDK used only for Graal, which requiries a very specific
+# version of the JDK. Too old, or too new will not work.
+BOOT_JAVA_HOME=${wrkdir}/jdk8u111-b14_fullsource/build/linux-x86_64-normal-server-release/images/j2sdk-image/
+BOOT_JDK_BASE=jdk8u111-b14
+BOOT_JDK_TAR=${BOOT_JDK_BASE}_fullsource.tgz
+build_bootstrap_jdk() {
+    echo "\n===> Download and build graal bootstrap JDK8\n"
+    if [ -f ${BOOT_JAVA_HOME}/bin/javac ]; then return; fi
+
+    cd ${wrkdir} || exit $?
+    # We fetch a hand-rolled tarball, as the JDK repo build downloads things
+    # and I am not sure that they are fixed versions. The tarball was rolled on
+    # 20161102.
+    if [ ! -f ${wrkdir}/${BOOT_JDK_TAR} ]; then
```

Comment:
I'm not sure that date is correct? It says November, but I suspect December is correct? Also, can we use ISO format YYYY-MM-DD pretty please? :)